documentation/src/main/asciidoc/topics/ref_endpoints_authentication_mechanisms.adoc

```diff
@@ -4,6 +4,11 @@
 {brandname} Server automatically configures endpoints with authentication mechanisms that match your security realm configuration.
 For example, if you add a Kerberos security realm then {brandname} Server enables the `GSSAPI` and `GS2-KRB5` authentication mechanisms for the Hot Rod endpoint.
 
+[IMPORTANT]
+====
+You cannot use the Lightweight Directory Access Protocol (LDAP) protocol with the DIGEST or SCRAM authentication mechanisms because of compatibility issues with the `UserPasswordCredentialLoaderBuilder` class.
```

Comment:
Users do not care about the class. I would just say.

"Currently, you cannot use... because those mechanisms need to have access to specific hashed passwords."

Reply:
Thanks, @tristantarrant . Updated. You OK to approve?